common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -150,6 +150,12 @@ class VideoFields(object):
         display_name=_("Upload Handout"),
         scope=Scope.settings,
     )
+    only_on_web = Boolean(
+        help=_("Specify whether this video is available only on the web and not other platforms like mobile apps."),
```

Comment:
Specify whether access to this video is limited to browsers only, or if it can be accessed from other applications including mobile apps.